src/main/java/com/tstepnik/planner/domain/Importance.java

```diff
@@ -0,0 +1,7 @@
+package com.tstepnik.planner.domain;
+
+public enum Importance {
+    IMPORTANT,
+    NORMAL,
+    INCIDENTAL;
```

Comment:
Bad name, doesn't fit here. google translate does pretty good job, just checked.

Reply:
I used diki dictionary, is better than google translate. Why "not_important" is better ?

Reply:
not important is bad name but google gave me better names. "minor" for example

Reply:
Ok so why minor is better than incidental ? :>

Reply:
because incidental means completely different thing and doesn't fit for our case.

Reply:
https://www.etutor.pl/profil#/slownik-angielskiego?q=incidental&popupmode=etutor

Reply:
Still, after looking at the link "incidental" doesn't make sense to me. You can use it if you want. It's your project, but I wouldn't do it.

Reply:
What is the decision here?

Reply:
minor